HeterogeneousCore/AlpakaCore/interface/alpaka/ESDeviceProduct.h

```diff
@@ -0,0 +1,52 @@
+#ifndef HeterogeneousCore_AlpakaCore_interface_alpaka_ESDeviceProduct_h
+#define HeterogeneousCore_AlpakaCore_interface_alpaka_ESDeviceProduct_h
```

Comment:
This file is one of those which should not really be in `HeterogeneousCore/AlpakaCore` because of the package depending on `FWCore/Framework`, it depends only on Alpaka itself, but is specific to CMS (so would not really fit in `HeterogeneousCore/AlpakaInterface`).